doc/api/deprecations.md

```diff
@@ -3056,6 +3056,23 @@ This method was deprecated because it is not compatible with
 
 Use [`buffer.subarray`][] which does the same thing instead.
 
+### DEP0159: Deprecated `_getActiveRequestsInfo` and `_getActiveHandlesInfo`
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/41587
+    description: Documentation-only deprecation.
+-->
+
+Type: Documentation-only
+
+The `process._getActiveHandlesInfo()` and `process._getActiveRequestsInfo()`
+is not intended for public use and can be removed in future releases.
+
+Use `process.getActiveResourcesInfo()` to get the list of handles and not
+the actual references.
```

Comment:
```suggestion
Use [`process.getActiveResourcesInfo()`][] to get a list of types of the active
resources instead of the actual references.
```
because resources = handles + requests here and you can add this line:
```md
[`process.getActiveResourcesInfo()`]: process.md#processgetactiveresourcesinfo 
```
after https://github.com/nodejs/node/blob/ccb8aae3932c13f33622203b2ffc5a33120e9d40/doc/api/deprecations.md#L3129, so that anyone can click the link and read more about the function.

Reply:
Updated